assets/app/views/newfromtemplate.html

```diff
@@ -0,0 +1,44 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-12">
+      <div ng-controller="NewFromTemplateController" class="create-from-template">
+        <div class="row">
+          <div class="col-md-3 template-name">
+            <span class="fa fa-cubes hidden-xs hidden-sm"></span>
+          </div>
+          <div class="col-md-7">
+            <h1><span class="fa fa-cubes visible-xs-inline visible-sm-inline"></span>{{ template.metadata.name }}</h1>
+            <div class="metadata">
+              <span ng-show="template.metadata.annotations.version">Version {{ template | annotation:'version' }}</span>
+              <span ng-show="template.metadata.annotations.provider">Provided By: {{ template | annotation:'provider' }}</span>
```

Comment:
I'll create a directive that contains that logic, and for consistency one for version too.